types/lightpick/index.d.ts

```diff
@@ -0,0 +1,384 @@
+// Type definitions for Lightpick 1.3
+// Project: https://wakirin.github.io/Lightpick
+// Definitions by: Adam Kwiatek <https://github.com/akwiatek>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+import * as moment from 'moment';
```

Comment:
Generally speaking, you can't instantiate a symbol imported in this way.  Maybe you meant `import moment = require('moment');`?

Reply:
Done.